.gitlab-ci/packet.yml

```diff
@@ -145,10 +148,10 @@ packet_almalinux8-calico-remove-node:
 packet_rockylinux9-calico:
   extends: .packet_pr_extended
 
-packet_almalinux8-calico:
+packet_almalinux9-calico:
   extends: .packet_pr_extended
 
-packet_almalinux8-docker:
+packet_almalinux9-docker:
   extends: .packet_pr_extended
```

Comment:
Should we add the label `ci-extend` to this PR?

Reply:
Good Idea